### PR TITLE
Added documentation for status notification schedule

### LINKED ISSUE #434: Documentation for status summary
We're missing some docs on how to setup the monika summary.

### PR DESCRIPTION
**Pull request by armedi**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. resolves #434

### CODE REVIEW

**Review by samsunoor — COMMENTED** [review]
Isn't it lacking explanation to disable the summary altogether?
AFAIK we can disable it with `--status-notification false`

**Review by sapiderman — APPROVED** [review]
thanks

File: docs/src/pages/guides/status-notification.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: status-notification
+title: Status Notification
+---
+
+Monika sends status notification based on a schedule with following information:
```
Comment (nicnocquee) [inline docs/src/pages/guides/status-notification.md:6]: Monika sends status notification periodically

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Comment by armedi** [comment]
> Isn't it lacking explanation to disable the summary altogether?
> AFAIK we can disable it with `--status-notification false`

oh yes, thank you for reminding me. I forgot about this

**Review by samsunoor — APPROVED** [review]

File: docs/src/pages/guides/status-notification.md (hunk 1 above)
Comment (armedi) [reply docs/src/pages/guides/status-notification.md]: updated in https://github.com/hyperjumptech/monika/pull/436/commits/46c8e3f73eb09beceea103feb1c941afc4be6a3f

**Comment by armedi** [comment]
> Isn't it lacking explanation to disable the summary altogether?
> AFAIK we can disable it with `--status-notification false`

updated in https://github.com/hyperjumptech/monika/pull/436/commits/46c8e3f73eb09beceea103feb1c941afc4be6a3f

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**